plugins/essif-lab_contactform7/admin/class-essif-lab_contactform7-admin.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * The admin-specific functionality of the plugin.
```

Comment:
Weet niet of het goed is om zoveel comments neer te zetten, ben er zelf geen voorstander van. Wat mij betreft maken de comments in dit bestand de paar regels aan code juist minder leesbaar. Kortom ik stem om dit soort fratsen voor nu gewoon weg te laten.